valexa/core/xlsx.py

```diff
@@ -43,3 +51,131 @@ def __get_data(self, sheet_name):
             data.append(row_values)
 
         return data
+
+class XlsxWriter:
+
+    def __init__(self, profile_to_use):
+        self.profile = profile_to_use
+
+    def write_profile(self, file_name):
+        workbook_file_name = file_name + '.xlsx'
+        if not self.__valid_if_file_exist(workbook_file_name):
+            working_workbook = Workbook()
+            working_workbook.save(workbook_file_name)
+
+        working_workbook = openpyxl.load_workbook(workbook_file_name)
+
+        if not self.profile.model.name  in working_workbook.sheetnames:
+            working_workbook.create_sheet(self.profile.model.name)
+
+        working_sheet = working_workbook[self.profile.model.name]
+        working_sheet['A1'] = self.profile.name_of_file
+        working_sheet['A2'] = 'Limit of Quantification'
+        working_sheet['B2'] = self.profile.min_lq
+        working_sheet.add_image(Image('Linear fig.png'), 'A3')
+
+        working_workbook.save(workbook_file_name)
+
+
+
+    def __valid_if_file_exist(self, file_name):
+        return path.exists(file_name)
+
+class HtmlWriter:
+
+    def __init__(self, profile_to_use):
+        self.profile = profile_to_use
+
+    def write_profile(self, file_name):
```

Comment:
the following logic seems to be linked to the HTML exportation. Should we move this outside the xlsx module? I think so.